internal/pkg/manifest/svc_test.go

```diff
@@ -60,7 +60,9 @@ environments:
 				wantedManifest := &LoadBalancedWebService{
 					Service: Service{Name: aws.String("frontend"), Type: aws.String(LoadBalancedWebServiceType)},
 					LoadBalancedWebServiceConfig: LoadBalancedWebServiceConfig{
-						Image: ServiceImageWithPort{ServiceImage: ServiceImage{Build: aws.String("frontend/Dockerfile")}, Port: aws.Uint16(80)},
+						Image: ServiceImageWithPort{ServiceImage: ServiceImage{Build: BuildArgsOrString{
```

Comment:
What do you think of adding tests for `BuildArgsOrString` that makes sure that when we call:
```go
testData := []byte(<single value or a map>)
var out BuildArgsOrString
yaml.Unmarshal(testData, &out)
// check if out has the right stuff
```

Same for MarshalYAML